### PR TITLE
Hotfix: fixing opearot job execution issues on mainnet

### PR DESCRIPTION
**Pull request by ACC01ADE**
## Describe Changes

Fixing mainnet operator issues with executing jobs fast and properly.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] I have checked eslint

### CODE REVIEW

File: src/utils/gas.ts [hunk 1]
```diff
@@ -117,6 +117,12 @@ export function updateGasPricing(
       gasPricing.maxFeePerGas = gasPricing.nextBlockFee!.add(gasPricing.nextPriorityFee!)
       gasPricing.gasPrice = gasPricing.maxFeePerGas
     }
+  } else {
+    if (gasPricing.gasPrice === null) {
```
Comment (alexanderattar) [inline src/utils/gas.ts:125]: can the function below return null?

Comment (alexanderattar) [inline src/utils/gas.ts:125]: When you make the update with more verbose logging can you also just add comment that this is for EIP1559

Comment (alexanderattar) [reply src/utils/gas.ts:125]: Shouldn't get triggered on any network except those that don't support EIP1559

Comment (ACC01ADE) [reply src/utils/gas.ts:125]: yep

**Review by sogoiii — APPROVED** [review]
👍

**Review by alexanderattar — APPROVED** [review]

**Pull request merged**